projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/requested-delivery-date/requested-delivery-date.ts

```diff
@@ -22,7 +22,7 @@ import {
 import { waitForPage } from '../../checkout-flow';
 import * as alerts from '../../global-message';
 
-export const poNumber = '777';
+export const poNumber = '111';
```

Comment:
`query:` Is this change related to the task?

Reply:
Yes, will refactor the variable into the corresponding ts file.